lib/screens/IngredientScreen.dart

```diff
@@ -618,3 +830,680 @@ class _IngredientPageState extends State<IngredientPage> {
+                    Text(
+                      'Ingredients',
+                      style: bottomRowOnScreenTextStyle,
+                      textAlign: TextAlign.center,
+                    )
+                  ],
+                ),
+              ),
+              Expanded(
+                flex: 2,
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.center,
+                  children: <Widget>[
+                    IconButton(
+                      onPressed: () {
+                        Navigator.restorablePushReplacementNamed(
+                            context, '/recipe');
+                      },
+                      icon: const Icon(Icons.restaurant),
+                      iconSize: bottomIconSize,
+                      color: bottomRowIcon,
+                    ),
+                    Text(
+                      'Recipes',
+                      style: bottomRowIconTextStyle,
+                      textAlign: TextAlign.center,
+                    )
+                  ],
+                ),
+              ),
+              Expanded(
+                flex: 2,
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.center,
+                  children: <Widget>[
+                    IconButton(
+                      onPressed: () {
+                        Navigator.restorablePushReplacementNamed(
+                            context, '/cart');
+                      },
+                      icon: const Icon(Icons.shopping_cart),
+                      iconSize: bottomIconSize,
+                      color: bottomRowIcon,
+                    ),
+                    Text(
+                      'Shopping Cart',
+                      style: bottomRowIconTextStyle,
+                      textAlign: TextAlign.center,
+                    )
+                  ],
+                ),
+              ),
+              Expanded(
+                flex: 2,
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.center,
+                  children: <Widget>[
+                    IconButton(
+                      onPressed: () {
+                        Navigator.restorablePushReplacementNamed(
+                            context, '/user');
+                      },
+                      icon: const Icon(Icons.person),
+                      iconSize: bottomIconSize,
+                      color: bottomRowIcon,
+                    ),
+                    Text(
+                      'User Profile',
+                      style: bottomRowIconTextStyle,
+                      textAlign: TextAlign.center,
+                    )
+                  ],
+                ),
+              ),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+
+//TODO(17): Attach API to get specified ingredient information
+//
+// List fetchIngredient(int ID) {
+//   ingredientInfo = a json of some sort;
+// }
+//
+}
```

Comment:
I think you have this function duplicated above, can remove one of the two

Reply:
The functions were copy-pasted, but they are in two different classes. Their actual functionality will ultimately be different once they are complete, but i put them their as a reminder for myself